server/setup.go

```diff
@@ -50,13 +50,11 @@ func Setup() error {
 		return err
 	}
 	cs := newCribbageServer(dbFactory)
-	err = seedNPCs(ctx, dbFactory)
-	if err != nil {
+	if err := seedNPCs(ctx, dbFactory); err != nil {
 		return err
 	}
-	cs.Serve()
 
-	return nil
+	return cs.serve()
```

Comment:
:t-rex: previously, if `cs.Serve()` errored, we would log and swallow. Now we return. That's probably better. :+1: